src/pykeen/pipeline/api.py

```diff
@@ -915,6 +915,15 @@ def pipeline(  # noqa: C901
     if evaluator_kwargs is None:
         evaluator_kwargs = {}
     evaluator_kwargs = dict(evaluator_kwargs)
+    if evaluator_kwargs.get('filtered'):
+        if use_testing_data and validation is not None:
+            logging.info(
+                "Validation triples are added to the set of known positive triples which are filtered out"
```

Comment:
this message is confusing. could you try rewording it?

Reply:
Done in 17b8bd850587e260637d77ac0f199ed227b3a579.